File: historian-tree/src/main/java/net/yslibrary/historian/tree/HistorianDebugTree.java
package net.yslibrary.historian.tree;

import timber.log.Timber;

public class HistorianDebugTree extends Timber.DebugTree {
  @Override
  protected String createStackElementTag(StackTraceElement element) {
    return String.format("(%s:%s) #%s ",
        element.getFileName(),
        element.getLineNumber(),
        element.getMethodName());
  }
}

File: historian-tree/src/main/java/net/yslibrary/historian/tree/HistorianDebugTree.kt
package net.yslibrary.historian.tree

import timber.log.Timber

class HistorianDebugTree : Timber.DebugTree() {
    override fun createStackElementTag(element: StackTraceElement): String {
        return String.format("(%s:%s) #%s ",
                element.fileName,
                element.lineNumber,
                element.methodName)
    }
}